### ISSUE TITLE
langchain rephrased the human input to a completely different meaning in the prompts

### ISSUE THREAD

**Issue by hpwahyao** [langchain-ai/langchain#6152]
### System Info

Recently, we do a few test on the ConversationalRetrievalChain + Memory, and notice that, the customer's question is rephrased by Langchain to a totally different meaning. Anyone knows how to avoid this?

The conversation is to introduce the different package of a mobile plan to the customer. and Customer inputs: **Hi**. However, while it comes to Langchain, this sentence is rephrased to: **What is your estimated monthly usage for data, talktime, and SMS?**

**Code:** 

qa = ConversationalRetrievalChain.from_llm(
            llm=self.llm, retriever=retriever, combine_docs_chain_kwargs={'prompt': self.QA_PROMPT}, verbose=True)    
result = qa({'question': question, 'chat_history': message_history.messages})

**Debugging infö as below:** 

Prompt after formatting:
Given the following conversation and a follow up question, rephrase the follow up question to be a standalone question, in its original language.

Chat History:

Human: I would like to have a data plan
Assistant: Sure, we have several data plans available. Can you please let me know your estimated monthly usage for data, talktime, and SMS? This will help me recommend the most suitable plan for you.
Follow Up Input: hi
Standalone question:

> Finished chain.

Human: I would like to have a data plan
Assistant: Sure, we have several data plans available. Can you please let me know your estimated monthly usage for data, talktime, and SMS? This will help me recommend the most suitable plan for you.
        
Human: What is your estimated monthly usage for data, talktime, and SMS?

### Information

- [ ] The official example notebooks/scripts
- [X] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [ ] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [X] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [X] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

Create a context of the conversation, and send some subjective sentence. 

### Expected behavior

Langchain rephrase it to a question.

**Comment by devstein** [comment]
_Generated by a 🤖_

### Answer

To avoid the issue of LangChain rephrasing the input to a completely different meaning, you can try adjusting the prompt used in the ConversationalRetrievalChain. You may need to experiment with different prompt structures to find one that works better for your specific use case. Additionally, you can try adjusting the temperature parameter of the language model to control the randomness of the generated output. Lower values will make the output more focused and deterministic, while higher values will make it more random.

### Sources
- [docs/modules/chains/index_examples/question_answering.ipynb](https://github.com/hwchase17/langchain/blob/master/docs/modules/chains/index_examples/question_answering.ipynb)

---

This response is meant to be useful and save users time. It is not meant to be a precise solution.

Please leave a 👍 if this is helpful and 👎 if it is irrelevant

**Comment by basyonic** [comment]
you can add specific instructions for input in tool description like the below:
"Company knowledgebase - useful for when you need to ask questions about company policies and news. The input to this tool should be a JSON object with one property 'query' with value of complete user input";

**Comment by zakone** [comment]
@hpwahyao  @basyonic 
 there is one param `condense_question_prompt`  in from_llm function, which will change the original question based on history, prompt like this
 
`"""Given the following conversation and a follow up question, rephrase the follow up question to be a standalone question.

Chat History:
{chat_history}
Follow Up Input: {question}
Standalone question:"""`

so I think you can modify the prompt to keep the question as same
